velox/functions/lib/DateTimeFormatter.h

```diff
@@ -218,10 +218,10 @@ class DateTimeFormatter {
 std::shared_ptr<DateTimeFormatter> buildMysqlDateTimeFormatter(
     const std::string_view& format);
 
-std::shared_ptr<DateTimeFormatter> buildJodaDateTimeFormatter(
+Expected<std::shared_ptr<DateTimeFormatter>> buildJodaDateTimeFormatter(
     const std::string_view& format);
 
-std::shared_ptr<DateTimeFormatter> buildSimpleDateTimeFormatter(
+Expected<std::shared_ptr<DateTimeFormatter>> buildSimpleDateTimeFormatter(
```

Comment:
It would be nice to extract this change (to make them return Expected) to a separate PR, and focus on changing 'DateTimeFunctions.h' in this PR. Thanks!

Reply:
@rui-mo Thanks for your suggestion. I've opened a separate PR in #11246 .